CHANGELOG.md

```diff
@@ -1,3 +1,146 @@
+# v1.13.0 (2022-03-15)
+
+## OS Changes
+
+* Set NVMe IO request timeouts for EBS according to AWS recommendations ([#2820])
+* Add `ethtool` to Bottlerocket ([#2829])
+* Create and set-up data partition at runtime ([#2807], [#2879], [#2873])
+* Add info logging to migrator in order to document onging migrations ([#2751])
+* Add `CAP_SYS_MODULE` and `CAP_CHROOT` to bootstrap containers ([#2772])
+* Add support for cgroup v2 ([#2875], [#2802])
+* Disable IA and SafeSetID LSM for kernel-5.15 ([#2789])
+* Update kernel-5.10 to 5.10.165 and kernel-5.15 to 5.15.90 ([#2795])
+* Allow `=` in bootconfig values ([#2806])
+* Include `systemd-analyze plot` for `logdog` ([#2880])
+* Update host containers ([#2864])
+* Upgrade `go-restful` code dependency used by containerd to avert unaffected CVE ([#2813])
```

Comment:
nit: this might be too minor a change to mention - I would just lump it in with the catch-all dependency updates for Go, Rust.